<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskMakerDialogBase</class>
 <widget class="QDialog" name="MaskMakerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Paleogeography - Mask preparation</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <widget class="QTabWidget" name="Tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Parameters</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>461</width>
              <height>579</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="2" column="0">
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Shallow seas</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Polygon layer*</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <item>
                     <widget class="QgsMapLayerComboBox" name="selectSsMask">
                      <property name="allowEmptyLayer" stdset="0">
                       <bool>false</bool>
                      </property>
                      <property name="showCrs" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="excludedProviders" stdset="0">
                       <stringlist>
                        <string>&quot;gdal&quot;</string>
                       </stringlist>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="selectSsButton">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Polyline  layer</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <item>
                     <widget class="QgsMapLayerComboBox" name="selectSsMaskLine">
                      <property name="allowEmptyLayer" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="showCrs" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="excludedProviders" stdset="0">
                       <stringlist>
                        <string>&quot;gdal&quot;</string>
                       </stringlist>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="selectSsLineButton">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Output file path:</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QgsFileWidget" name="outputPath" native="true"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Coastlines</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Polygon layer*</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <item>
                     <widget class="QgsMapLayerComboBox" name="selectCoastlineMask">
                      <property name="allowEmptyLayer" stdset="0">
                       <bool>false</bool>
                      </property>
                      <property name="showCrs" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="excludedProviders" stdset="0">
                       <stringlist>
                        <string>&quot;gdal&quot;</string>
                       </stringlist>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="selectCoastButton">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Polyline  layer</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <item>
                     <widget class="QgsMapLayerComboBox" name="selectCoastlineMaskLine">
                      <property name="allowEmptyLayer" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="showCrs" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="excludedProviders" stdset="0">
                       <stringlist>
                        <string>&quot;gdal&quot;</string>
                       </stringlist>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="selectCoastLineButton">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Continental shelves</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Polygon layer*</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <item>
                     <widget class="QgsMapLayerComboBox" name="selectCshMask">
                      <property name="allowEmptyLayer" stdset="0">
                       <bool>false</bool>
                      </property>
                      <property name="showCrs" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="excludedProviders" stdset="0">
                       <stringlist>
                        <string>&quot;gdal&quot;</string>
                       </stringlist>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="selectCsButton">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Polyline  layer</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <item>
                     <widget class="QgsMapLayerComboBox" name="selectCshMaskLine">
                      <property name="allowEmptyLayer" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="showCrs" stdset="0">
                       <bool>true</bool>
                      </property>
                      <property name="excludedProviders" stdset="0">
                       <stringlist>
                        <string>&quot;gdal&quot;</string>
                       </stringlist>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="selectCsLineButton">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="4" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="helpBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>320</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Help shall be displayed here.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextBrowser" name="logText">
         <property name="autoFormatting">
          <set>QTextEdit::AutoAll</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="warningLabel">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MaskMakerDialogBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
